config.json

```diff
@@ -824,7 +824,12 @@
         "name": "Matching Brackets",
         "uuid": "4d456646-3a9b-4393-9558-6b30e5c1039c",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "strings",
+          "arrays",
+          "conditionals-switch",
+          "for-loops"
+        ],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/matching-brackets/solutions/TomPradat

I've seen many solutions with conditional switches and I prefer it to regexps ones. Also I've added arrays because I think this is good exercise for 'stack' type solutions